src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -65,6 +86,50 @@ case class WorkflowDescriptor(id: WorkflowId,
     case _ => NOPLogger.NOP_LOGGER
   }
 
+  def postProcessWorkflow(implicit executionContext: ExecutionContext): Future[Any] = {
+    // Try to copy outputs to final destination
+    wfOutputsRoot map copyOutputFiles getOrElse Future.successful({})
+  }
+
+  private def copyOutputFiles(destDirectory: String)(implicit executionContext: ExecutionContext): Future[Unit] = {
+    import PathString._
+    val logger = backend.workflowLogger(this)
+
+    def copyFile(file: WdlFile): Try[Unit] = {
+      val src = file.valueString.toPath(gcsFilesystem)
+      val wfPath = wfContext.root.toPath(gcsFilesystem).toAbsolutePath // Absolute path of the workflow directory
```

Comment:
I don't think the comment really adds anything, the variable is `wfPath` and the line ends with `toAbsolutePath`. If you did want to be pedantic just call it `workflowPath`